src/Presentation/Nop.Web/Views/Checkout/BillingAddress.cshtml

```diff
@@ -37,6 +37,22 @@
                             </div>
                         </div>
                     }
+                    @if (Model.GuestCustomerVatEnabled || Model.DisplayGuestCustomerVatWarning)
+                    {
+                        <div class="section vat-number">
+                            @if (Model.DisplayGuestCustomerVatWarning)
+                            {
+                                <label class="vat-number-warning">
+                                    @T("Checkout.VatNumber.Warning")
+                                </label>
+                            }
+                            @if (Model.GuestCustomerVatEnabled)
+                            {
+                                <label asp-for="VatNumber">@T("Checkout.VatNumber")</label>

```

Comment:
Let's always display this label, and if the setting is disabled, then show a warning alongside.